ChameleonForms.Example/Controllers/ExampleFormsController.cs

```diff
@@ -209,25 +229,32 @@ public ViewModelExample()
         public bool Boolean { get; set; }
 
         public ChildViewModel Child { get; set; }
+
+        [DisplayFormat(DataFormatString = "{0:g}", ApplyFormatInEditMode = true)]
+        public DateTime DateField { get; set; }

```

Comment:
Hmm, this is less important than I thought - I originally thought it was in the test project.